CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ## x.y.z
 
 ### Added
+- changes for multidomainsetup: TCA, Flexforms, pageTS, documentation
```

Comment:
Please add a link to the PR ID.

Also, I suggest a wording change. I propose: `Multi-domain setup`